src/backend/distributed/worker/worker_data_fetch_protocol.c

```diff
@@ -679,7 +683,7 @@ ExtractShardId(StringInfo tableName)
  * false. On other types of failures, the function errors out.
  */
 static bool
-FetchRegularTable(const char *nodeName, uint32 nodePort, StringInfo tableName)
+FetchRegularTable(const char *nodeName, uint32 nodePort, const char *schemaName, const char * tableName)
```

Comment:
This exceeds our line limit. Do you have [`citus_indent`](https://github.com/citusdata/tools/tree/v0.3.2/uncrustify) installed?